srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditAbstractNumericalVar.java

```diff
@@ -0,0 +1,87 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import com.google.common.collect.*;
+
+import gov.va.med.srcalc.domain.calculation.ValueRetriever;
+import gov.va.med.srcalc.domain.model.*;
+import gov.va.med.srcalc.service.ModelInspectionService;
+
+/**
+ * Provides common functionality for editing an {@link AbstractNumericalVariable}.
+ */
+public abstract class EditAbstractNumericalVar extends EditBaseVar
+{
+    private static final ImmutableSortedSet<ValueRetriever> RETRIEVERS;
+    static
+    {
+        // Kludge: all the retrievers happen to be numerical except for gender,
+        // so use that fact to build the list of numerical retrievers.
+        final HashSet<ValueRetriever> tempRetrievers =
+                new HashSet<>(Arrays.asList(ValueRetriever.values()));
+        tempRetrievers.remove(ValueRetriever.GENDER);
+        RETRIEVERS = ImmutableSortedSet.copyOf(Ordering.usingToString(), tempRetrievers);
+    }
+
+    private String fUnits;
+    private final NumericalRangeBuilder fValidRange;
+    
+    /**
+     * Constructs an instance with default values for all properties.
+     * @param modelService to provide reference data (e.g., available
+     * VariableGroups) to the user
+     */
+    public EditAbstractNumericalVar(final ModelInspectionService modelService)
+    {
+        super(modelService);
+        
+        fUnits = "";
+        fValidRange = new NumericalRangeBuilder();
+    }
+    
+    @Override
+    public ImmutableSortedSet<ValueRetriever> getAllRetrievers()
```

Comment:
This new method isn't used anywhere, so what's the point of doing the sorting earlier and providing this method? Is it going to be used for editing a discrete numerical?